lib/stripe/errors.rb

```diff
@@ -3,12 +3,24 @@ module Stripe
   # errors derive.
   class StripeError < StandardError
     attr_reader :message
-    attr_reader :http_status
+
+    # Response contains a structure that has some basic information about the
+    # response that conveyed the error.
+    attr_accessor :response
+
+    # These fields are now available as part of #response and that usage should
+    # be preferred.
     attr_reader :http_body
     attr_reader :http_headers
+    attr_reader :http_status
+    attr_reader :json_body # equivalent to #data
```

Comment:
I'm thinking it might be a good idea to deprecate the use of these fields and make them pass throughs to `response` in the meantime, but I deferred that for the time being.